src/main/java/seedu/address/ui/PriorityIndicatorComponent.java

```diff
@@ -0,0 +1,107 @@
+package seedu.address.ui;
```

Comment:
Can you add the updated diagram in the diagrams PR? So we don't forget

Reply:
ah ok i realised i clda just... edited it LOL